admin_guide/overcommit.adoc

```diff
@@ -112,6 +112,26 @@ system tasks or daemons need more memory than was accounted for in the node's
 resource reservation. If a container specifies a limit on memory, it is
 immediately killed if it exceeds the limit amount.
 
+[[overcommit-ephemeral-storage]]
+=== Ephemeral Storage
+
+This applies only if you have enabled the Ephemeral Storage tech
+preview in {product-title} 3.10.  This is disabled by default; to
+enable this feature, please see
+xref:../install_config/configuring_ephemeral.adoc[Configuring for
+Ephemeral Storage].
+
+A container is guaranteed the amount of ephemeral storage it
```

Comment:
Is this describing how ephemeral storage works with containers?

Reply:
It describes what happens if your limit is higher than your request -- it's parallel to the CPU and memory sections immediately above.

Reply:
Resolved.